marshmallow_jsonapi/schema.py

```diff
@@ -277,6 +277,17 @@ def inflect(self, text):
 
     ### Overridable hooks ###
 
+    def _extract_error_from_dict(self, field_name, field_errors):
+        if field_errors and isinstance(field_errors, dict):
+            k, field_errors = list(field_errors.items()).pop()
```

Comment:
This doesn't look quite right. What if there are more than one field that contains errors?

Reply:
Sorry for the delay. It has been very intense at work this last days.
I will take a look at it and see if I can address the issues you pointed out.
Thanks.

Reply:
I've already resolved these in #289